pyensign/exceptions.py

```diff
@@ -35,6 +133,14 @@ class EnsignResponseType(EnsignError):
     pass
 
 
+class EnsignAttributeError(EnsignError):
```

Comment:
I'd recommend that this subclass both `EnsignError` and `AttributeError`

```suggestion
class EnsignAttributeError(EnsignError, AttributeError):
```